components/ws-daemon/cmd/run.go

```diff
@@ -43,85 +38,34 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			log.WithError(err).Fatal("cannot read configuration. Maybe missing --config?")
 		}
+
 		reg := prometheus.NewRegistry()
+		reg.MustRegister(
+			collectors.NewGoCollector(),
+			collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
+		)
```

Comment:
We should move this into baseserver just like in https://github.com/gitpod-io/gitpod/pull/9973/files#diff-7d515f84aa53501b0b1dd02e94b43be371f6505b57ae62490c39342b0b994d39R368-R380

Reply:
Moved this part into a separate PR: https://github.com/gitpod-io/gitpod/pull/10050

Reply:
Moved this part into a separate PR: https://github.com/gitpod-io/gitpod/pull/10050